honeybee_vtk/actors.py

```diff
@@ -0,0 +1,154 @@
+"""Vtk actors that are added to a vtk scene."""
+
+import vtk
+from .model import Model, ModelDataSet, DisplayMode
+from .types import JoinedPolyData
+from ._helper import _check_tuple
+from ladybug_geometry.geometry3d.pointvector import Point3D
+
+
+class Actors:
+    """Create vtk actors from Model.
+
+    Objects in Honeybee such as walls, floors, ceilings, shades, apertures, rooms are
+    called actors in vtk terms. The actors are created from the Modeldatasets in a
+    Model object.
+
+    Args:
+        model: A Model object created using honeybee-vtk Model.
+            Defaults to None.
+    """
+    def __init__(self, model):

```

Comment:
After checking the rest of the code I think the input here shouldn't be a model. It should be a `ModelDataSet` - then you can have a `classmethod` to create this class `from_model`.

Reply:
Complied.